IPDatePicker/Classes/InfiniteTableView.swift

```diff
@@ -0,0 +1,333 @@
+//
+//  InfiniteTableView.swift
+//  Pods
+//
+//  Created by Andrew Dolce on 9/2/17.
+//
+//
+
+import UIKit
+import PureLayout
+
+protocol InfiniteTableViewDataSource: class {
+    func numberOfItems(in infiniteTableView: InfiniteTableView) -> Int
```

Comment:
I commented on this method name where you implement it, not in the protocol definition.  Oops.

Reply:
Never mind, I see what you're doing here.